flink-core/src/test/java/org/apache/flink/api/common/io/FileOutputFormatTest.java

```diff
@@ -44,7 +44,7 @@ public void testCreateNoneParallelLocalFS() {
 		} catch (IOException e) {
 			throw new RuntimeException("Test in error", e);
```

Comment:
We do this in other tests as well, but the test will fail if it throws an Exception anyways, so it's OK to just declare it in the signature instead of rethrowing.